lib/sidekiq/instrument/worker.rb

```diff
@@ -21,18 +21,22 @@ def perform
       self.class::METRIC_NAMES.each do |method, stat|
         stat ||= method
 
-        StatsD.gauge "shared.sidekiq.stats.#{stat}", info.send(method)
+        Statter.statsd.gauge("shared.sidekiq.stats.#{stat}", info.send(method))
+        Statter.dogstatsd&.gauge("sidekiq.#{stat}", info.send(method))
```

Comment:
you may have to use the fill name of the class with module ```Sidekiq::Instrument::Statter`

Reply:
It seems to work without it, but I might be off in my understanding: I thought if we were in `module Sidekiq::Instrument` (line 4 of [lib/sidekiq/instrument/worker.rb](https://github.com/enova/sidekiq-instrument/pull/8/files#diff-96ee20da218e9ee0b2400dc57d72dcc38ae0f07402329892698358514aa43b10)) we could access other definitions from that namespace without the full prefix.